src/sessions.ts

```diff
@@ -189,21 +241,25 @@ export class ClientSession extends TypedEventEmitter<ClientSessionEvents> {
    */
   endSession(): Promise<void>;
   endSession(callback: Callback<void>): void;
-  endSession(options: Record<string, unknown>): Promise<void>;
-  endSession(options: Record<string, unknown>, callback: Callback<void>): void;
+  endSession(options: EndSessionOptions): Promise<void>;
+  endSession(options: EndSessionOptions, callback: Callback<void>): void;
   endSession(
-    options?: Record<string, unknown> | Callback<void>,
+    options?: EndSessionOptions | Callback<void>,
     callback?: Callback<void>
   ): void | Promise<void> {
     if (typeof options === 'function') (callback = options), (options = {});
-    options = options ?? {};
+    const _options = options ?? ({} as EndSessionOptions);
+    if (_options.force == null) _options.force = true;
```

Comment:
```suggestion
    if (typeof options === 'function') {
      callback = options;
    }
    const finalOptions = { force: true, ...options };
```

We have some duplication of defaulting the options to an empty object here, and we can combine the defaulting of force. I suggest finalOptions here, we've just used that pattern elsewhere. 

Why is force default true now?

Reply:
Changed to use `finalOptions` pattern here now. As for forcing `true` to default, ending the session should always unpin the connection in the default case, even out of lb mode. (Since it won't ever be pinned anyway and would be a no-op). The only thing we ever want to specify is when *not* to unpin.